adam-cli/src/main/scala/org/bdgenomics/adam/cli/FlagStat.scala

```diff
@@ -121,7 +121,8 @@ class FlagStat(protected val args: FlagStatArgs) extends BDGSparkCommand[FlagSta
         val fs = FileSystem.get(conf)
         val path = new Path(outputPath)
         val outputStream = fs.create(path, true)
-        outputStream.writeUTF(output)
+        outputStream.writeBytes(output)
```

Comment:
fwiw I just tested with `writeChars` and my shell displays the results just like it did before, which was my only litmus test, so I'm fine either way